setup.py

```diff
@@ -0,0 +1,4 @@
+from setuptools import setup
```

Comment:
This is another file that makes the python package installable.

Reply:
The `zppy_e3sm_chem_diags/__init__.py` is the final file that makes this a python package.  It is currently empty but it is required.  You can add things to it later.  We can talk about why you might want to do that.  (It can be used to simplify imports from this package, e.g. within `zppy`.)

Reply:
Each directory (e.g. `templates`) within the package also needs a `__init__.py` so it is included in the package.  Keep that in mind if you add new directories within the package.

Also, as I noted above, all files outside of `zppy_e3sm_chem_diags` are not considered to be part of the python package.

Reply:
Got it. `__init__.py` is needed for each directory in the package.

I assume the information (except format) should be consistent between `setup.cfg` and `setup.py`, which is handled manually.

Reply:
Yes, the same information is needed in either `setup.py` or `setup.cfg`.  Either place is fine.  This package is pretty simple so I think `setup.cfg` will be easier.

Reply:
Okay, let's just use `setup.cfg`.

Reply:
I don't feel very comfortable with `pyproject.toml` files yet, which is why I didn't go for that solution.  I have more trouble maintaining packages on conda-forge that use that format than the older approaches (though that has more to do with grayskull not supporting the `pyproject.toml` format).  Still, if someone wants to give that a go, that's fine.